src/Microsoft.DotNet.Darc/src/DarcLib/Models/VirtualMonoRepo/AllVersionsPropsFile.cs

```diff
@@ -36,21 +36,43 @@ public AllVersionsPropsFile(Dictionary<string, string> versions)
         Versions = versions;
     }
 
-    public (string? Sha, string? Version) GetVersion(string repository)
+    public (string? Sha, string? Version)? GetVersion(string repository)
     {
         var key = SanitizePropertyName(repository) + ShaPropertyName;
         Versions.TryGetValue(key, out var sha);
 
         key = SanitizePropertyName(repository) + PackageVersionPropertyName;
         Versions.TryGetValue(key, out var version);
+
+        if (sha is null && version is null)
+        {
+            return null;
```

Comment:
Out of curiosity, are there any guidelines to return null when using tuples in C#? I.e., null ofr each of the return values vs. using the null for the whole tuple.

Reply:
I have created an enclosing type for the tuple